net/wgsd/README.md

```diff
@@ -0,0 +1,41 @@
+## Wireguard Service Discovery (Mesh)
+
+This tool allows you to build a mesh from wireguard tunnels and also traversal NAT.
+
+See this article: https://www.jordanwhited.com/posts/wireguard-endpoint-discovery-nat-traversal/
+Also see project on github: https://github.com/jwhited/wgsd
+
+
+## Installation
+
+For Server
+
+    opkg install wgsd-coredns
+
+For Client
+
+    opkg install wgsd-client
```

Comment:
Do we really NEED this README? I dont think so. This is kinda obvious, how to install packages. If not, look at the documentation. The usage? The same. Create documentation in OpenWrt wiki for that.

Reply:
I've used wg-installer readme as a template.

Reply:
Still, it does not mean it is right. For that, there is OpenWrt wiki. I think it is quite obvious how to install package for client or server. On the other hand, we can have that for each package in OpenWrt and I dont think it makes sense.